src/common/strings.ts

```diff
@@ -0,0 +1,183 @@
+import LocalizedStrings, { LocalizedStringsMethods } from "react-localization";
+
+export interface IStrings extends LocalizedStringsMethods {
+    appName: string;
+    common: {
+        displayName: string;
+        description: string;
+        submit: string;
+        cancel: string;
+    };
+    homePage: {
+        newProject: string;
+        openProject: string;
+        recentProjects: string;
+    };
+    menuBar: {
+        file: {
+            exit: string;
+        }
+        edit: {
+            undo: string;
+            redo: string;
+            cut: string;
+            copy: string;
+            paste: string;
+            pasteAndMatchStyle: string;
+            delete: string;
+            selectAll: string;
+        }
+        view: {
+            reload: string;
+            forceReload: string;
+            toggleDevTools: string;
+            actualSize: string;
+            toggleFullScreen: string;
+        }
+        window: {
+            minimize: string;
+            close: string;
+        }
+        help: {
+            learnMore: string;
+            documentation: string;
+            communityDiscussions: string;
+            searchIssues: string;
+        },
+    };
+    projectSettings: {
+        sourceConnection: string;
+        targetConnection: string;
+        addConnection: string;
+        tags: string;
+    };
+    connections: {
+        title: string;
+        details: string;
+        settings: string;
+        connectionPageInstructions: string;
+        provider: string;
+        providers: {
+            azureBlob: {
+                title: string;
+            }
+            bing: {
+                title: string;
+                options: string;
+                apiKey: string;
+                query: string;
+                aspectRatio: string;
+            },
+            local: {
+                title: string;
+                folderPath: string;
+                selectFolder
+            },
+        }
+    };
+    editorPage: {
+        width: string;
+        height: string;
+        toolbar: {
+            select: string;
+            pan: string;
+            drawRectangle: string;
+            drawPolygon: string;
+            saveProject: string;
+            exportProject: string;
+        }
+    };
+}
+
+export const strings: IStrings = new LocalizedStrings({
```

Comment:
Can these language specific strings somehow be save as json files with some naming convention like `strings.en.json` and `strings.es.json` ?